### PR TITLE
join DAO & vote in one click

### LINKED ISSUE #131: Enter DAO and vote in one click
**Is your feature request related to a problem? Please describe.**
To improve the experience, we would like to make joining a DAO seamless for the user. Right now, the user is required to click twice: first to join a DAO and then to make his vote. We would like to make both actions with a single button click.

**Describe the solution you'd like**
The user that wants to vote on a proposal from a public treasury would be able to join and vote with a single click.
If the treasury is private, no button should be displayed.  The message `You can't join a private DAO` in the place of the button would help clarify why.

### PR DESCRIPTION
**Pull request by Viterbo**
# Description
if the user is not registered in the DAO, the button will perform both actions in a single click: "join DAO & vote"

Fixes #131

### CODE REVIEW

File: src/pages/trails/ballots/view/BallotView.vue [hunk 1]
```diff
@@ -218,35 +226,26 @@ export default {
...
         let register = false;
-        if (this.isUserRegisteredInTreasury) {
-            register = false;
-        } else {
-            // ---- quickfix for #92 -------
-            /*
-            if (this.ballot.treasury.access == 'public') {
-                register = true;
+
+        if (this.isPositiveVotePower) {
```
Comment (karynemayer) [inline src/pages/trails/ballots/view/BallotView.vue:232]: Treasuries that consider the liquid balance of the accounts can create ballots that disregard the staked balance for voting. [This issue](https://github.com/telosnetwork/app-telos-net/issues/44) was created to address the `votestake` and `voteliquid` behavior a while back.

I have created [this beauty](https://miro.com/app/board/uXjVPQ6EDvA=/) to map the flow. The opening tutorial in another tab is my suggestion for guiding the user to the next step.

**Review by karynemayer — CHANGES_REQUESTED** [review]

File: src/pages/trails/ballots/view/BallotView.vue (hunk 1 above)
Comment (Viterbo) [reply src/pages/trails/ballots/view/BallotView.vue:232]: @karynemayer, I just sent an Access request for the Miro site (using my normal email viter.rod@gmail.com)

Comment (karynemayer) [reply src/pages/trails/ballots/view/BallotView.vue:232]: @Viterbo I've updated the link and granted you editor access.

**Review by donnyquixotic — APPROVED** [review]

File: src/pages/trails/ballots/view/BallotView.vue (hunk 1 above)
Comment (Viterbo) [reply src/pages/trails/ballots/view/BallotView.vue:284]: In this commit [25ceabb](https://github.com/telosnetwork/app-telos-native/pull/171/commits/25ceabb7db186b24201787a8ce06943d023dc828), the official ballot cases (with `VOTE` symbol) are handled separately from those that are not. 

In the latter case (with custom symbol), it is also handled properly if the ballot is `votestake` and when it is `voteliquid`.

**Review by karynemayer — APPROVED** [review]

**Pull request merged**